src/tentaclio/credentials/reader.py

```diff
@@ -85,6 +90,25 @@ def _load_from_file(
         raise TentaclioFileError("File not found") from e
 
 
+def replace_env_variables(url: str) -> str:
```

Comment:
This should be a private function and therefore it should be called `_replace_env_variables`